go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/google/gnostic-models v0.6.9-0.20230804172637-c7be7c783f49
 	github.com/google/go-cmp v0.6.0
 	github.com/google/go-containerregistry v0.17.0
+	github.com/google/uuid v1.5.0
```

Comment:
Note: this is not an accident. `datastore/postgres/mocks/matcher_store` uses this